etl/grapher_helpers.py

```diff
@@ -263,6 +267,36 @@ def yield_long_table(
         yield from yield_wide_table(cast(catalog.Table, t), dim_titles=dim_titles)
 
 
+def long_to_wide_tables(
+    table: catalog.Table,
+    annot: Optional[Annotation] = None,
+) -> Iterable[catalog.Table]:
+    """Yield wide tables from long table with the following columns:
+    - variable: short variable name (needs to be underscored)
+    - value: variable value
+    - meta: either VariableMeta object or null in every row
```

Comment:
In the description, it's ambiguous whether the wide table or the long table has this column structure.

```
Takes as input a long table, which must have columns: ...

Yields one or more wide tables from it, which have one column per variable.
```